modules/ROOT/pages/view-clear-logs-task.adoc

```diff
@@ -1,26 +1,76 @@
-= View Logs
+= View the Log for a Mule App Project in Design Center
```

Comment:
@rddheath how about saying 'View the Log for a Mule App Project in Flow Designer'

Reply:
Oh, yeah. Because I'm using the name now in the parent topic for the FD docs, I can use it here, too.